### ISSUE TITLE
йцу

### ISSUE THREAD

**Issue by dialandrey** [avocode/avocode#1851]
---
* **Version**: 2.19.0-web.2
* **Web/Desktop**: Web
* **Platform**: windows 6.1
* **Browser**: chrome 64.0.3282.140
* **Avocode UID**: 266501
---
```
TypeError: Cannot read property 'width' of null
    at Object.h [as drawResultToCanvasScaled] (https://cdn.app.avocode.com/dist/v-7a9c360/js/avocode-web.min.js:17:150193)
    at n.preview (https://cdn.app.avocode.com/dist/v-7a9c360/js/avocode-web.min.js:60:62179)
    at e.value (https://cdn.app.avocode.com/dist/v-7a9c360/js/avocode-web.min.js:106:1365958)
    at t.describeLayers (https://cdn.app.avocode.com/dist/v-7a9c360/js/avocode-web.min.js:106:1408985)
    at e.value (https://cdn.app.avocode.com/dist/v-7a9c360/js/avocode-web.min.js:106:1637993)
    at e.value (https://cdn.app.avocode.com/dist/v-7a9c360/js/avocode-web.min.js:106:1636090)
    at t._getSelectionState (https://cdn.app.avocode.com/dist/v-7a9c360/js/avocode-web.min.js:106:459310)
    at t._handleSelectionChange (https://cdn.app.avocode.com/dist/v-7a9c360/js/avocode-web.min.js:106:458647)
    at t.<anonymous> (https://cdn.app.avocode.com/dist/v-7a9c360/js/avocode-web.min.js:106:456064)
    at r (https://cdn.app.avocode.co
```

**Comment by vojtatranta** [comment]
Hi @dialandrey ,
this happened when you tried to select a layer?

**Comment by dialandrey** [comment]
yes08.02.2018, 12:03, "Vojtěch Tranta" <notifications@github.com>:Hi @dialandrey ,
this happened when you tried to select a layer?

**Comment by vojtatranta** [comment]
OK, we will hopefully fix it in the comming `2.19.1`

**Comment by vojtatranta** [comment]
This should be fixed in `2.19.1`
